spec/support/monitor/monitor_idv_steps.rb

```diff
@@ -14,18 +14,27 @@ def verify_identity_with_doc_auth
     expect(page).to have_current_path('/verify/doc_auth/upload')
 
     click_on 'Upload from your computer'
-    expect(page).to have_current_path('/verify/doc_auth/front_image')
 
-    click_doc_auth_fallback_link
+    if current_path == '/verify/doc_auth/document_capture'
+      # React-based document capture flow is enabled
+      attach_file 'file-input-1', File.expand_path('spec/fixtures/ial2_test_credential.yml')
+      attach_file 'file-input-2', File.expand_path('spec/fixtures/ial2_test_credential.yml')
```

Comment:
The file input IDs aren't especially stable, since they're generated using an [incrementing instance ID counter](https://github.com/18F/identity-idp/blob/21be6a360593180a608324fb4b43306bf14ef9ed/app/javascript/packages/document-capture/components/file-input.jsx#L109).

If I'm reading [the `attach_file` documentation](https://rubydoc.info/github/jnicklas/capybara/master/Capybara%2FNode%2FActions:attach_file) correctly, could we use each field's label instead of the ID?

Reply:
Good point, will do

Reply:
Updated in 9ae6fe6a07ecb202154defe78d4d09f74a7ed60d